System.IO.Ports/SerialData.cs

```diff
@@ -11,12 +11,12 @@ namespace System.IO.Ports
     public enum SerialData
     {
         /// <summary>
-        /// A character was received and placed in the input buffer.
+        /// A character was received and placed in the input stream.
         /// </summary>
         Chars = 1,
 
         /// <summary>
-        ///  The end of file character was received and placed in the input buffer.
+        ///  The character to watch was received in the input buffer and prior content placed in the input stream.
```

Comment:
I'm not English native, but "(...) and prior content placed (...)" sounds confusing to me and I don't understand what it it means exactly. 

Edge case: only the watch char was received, there is no other content in the stream, so... My point being: we don't have to try to explain here every possible situation and workflow that could happen. For this one we have to say exactly and just that: the watch char was detected in the input stream. Period. Everything beyond that just adds confusion.